fpdcc/document-search
# Import new attributes and geometries for Licenses

## Overview

Import new attributes and geometries for Licenses using a database dump provided by the GIS team.

## Testing Instructions

* This PR adds new application requirements, so you'll have to rebuild your images:

```
docker-compose build
```

* This PR makes some changes to the Solr index, so you'll need to drop it before you start:
    * Make sure your network is down with `docker-compose down`
    * Remove the Solr volume with `docker volume rm document-search_solr-data`
* Redo the data import:

```
docker-compose -f docker-compose.yml -f data/docker-compose.yml run --rm app make all
```

* Run `docker-compose run --rm app ./manage.py rebuild_index` to rebuild the Solr index
* Next, run `docker-compose up` and test the interface:
    * Navigate to http://localhost:8000/licenses/search/
    * Confirm the new `Geometry` column shows maps
    * Try all of the new facets (everything except `License number`)
